packages/http/src/stamp.ts

```diff
@@ -23,16 +26,20 @@ export async function stamp(input: {
   };
 }
 
-async function importPrivateKey(
-  privateKeyHex: string
-): Promise<webcrypto.CryptoKey> {
-  const privateKeyPkcs8Der = hexStringToUint8Array(
-    PRIVATE_KEY_PREFIX + privateKeyHex
-  );
+async function importPrivateKey(input: {
```

Comment:
Now that this function takes both public and private key, I'd call it `importTurnkeyApiKey`